apps/arweave/src/ar_webhook.erl

```diff
@@ -29,10 +20,29 @@
 	{<<"content-type">>, <<"application/json">>}
 ]).
 
+-define(MAX_TX_OFFSET_CACHE_SIZE, 100_000). %% Max number of transactions cached
+
+%% Functions which update or query the #tx_offset_cache are named with the `cache_` prefix
+%% e.g. `cache_is_tx_data_marked_synced/2`.
+-record(tx_offset_cache, {
+	txid_to_timestamp = #{}, % Map TXID => Timestamp
+	timestamp_to_txid = gb_sets:new(), % Ordered set of {Timestatmp, TXID}
```

Comment:
Sorry :( I didn't mean to add `_to_` to the set names, just the map names (to differentiate key from value). I'm not sure how to improve the set names over what you had before (e.g. `timestamp_txid`) and I'm not sure adding `_to_` to the set names helps with readability.

Commenting here just as FYI. I"m also fine if you leave the `_to_` in there on the set names - we can update later if needed.

Reply:
No worries! I do not have a strong preference either, reverted

Reply:
Great!